ObjcVoiceQuickstart/ViewController.m

```diff
@@ -356,9 +356,10 @@ - (void)cancelledCallInviteReceived:(TVOCancelledCallInvite *)cancelledCallInvit
     NSLog(@"cancelledCallInviteReceived:");
     
     TVOCallInvite *callInvite;
-    for (TVOCallInvite *invite in self.activeCallInvites) {
-        if ([cancelledCallInvite.callSid isEqualToString:invite.callSid]) {
-            callInvite = invite;
+    for (NSString *activeCallInviteId in self.activeCallInvites) {
```

Comment:
```
for (NSString *callSid in self.activeCallInvites) {
    TVOCallInvite *activeCallInvite = [self.activeCallInvites objectForKey:callSid];
    ...
}
```

Reply:
I'm happy to change the var name. Just want to confirm we're all on the same page before choosing a different name.

If I am understanding this code correctly https://github.com/twilio/voice-quickstart-objc/blob/master/ObjcVoiceQuickstart/ViewController.m#L342 then it is _not_ the call SID, but a CallKit UUID. The UUID is defined in the `TVOCallInvite.h` as:

```
 * @discussion Use this UUID for CallKit methods. Accepting a `TVOCallInvite` yields a `TVOCall` which inherits its
 * UUID from the Invite.
```

Perhaps it's better to change the var name to `activeCallInviteUUID` or to change the dictionary key to the call SID on line 342?

Reply:
@lonnylot 
Sorry for my delay of responding back to your comment, and yes you are right, I was using the UUID as the key of the dictionary. In this case I think `for (NSString *uuid in self.activeCallInvites) { ... }` should be good enough.

Thanks again!